core/src/main/java/org/apache/accumulo/fate/ReadOnlyTStore.java

```diff
@@ -36,7 +36,9 @@
   enum TStatus {
     /** Unseeded transaction */
     NEW,
-    /** Transaction is eligible to be executing */
+    /** Transaction that is eligible to be executed */
+    SUBMITTED,
+    /** Transaction that is executing */
```

Comment:
New enums should be appended to the list rather than inserted into the middle, so it doesn't affect their ordinal, which could be important for serialization issues.

Reply:
Addressed in 46a503ace773ab8f3c1f35a91691c78014e43f9b